lib/DDG/Spice/PackageTracking.pm

```diff
@@ -74,6 +74,43 @@ triggers query_nowhitespace => qr/^
                                 )
                                 $/xi;
 
+## CanadaPost
+triggers query_nowhitespace_nodash => qr/^
+                                (?:
+                                    [\d]{9,} |
+                                    [A-Z]{2}\d{9}CA)
+                                )
+                                $/xi;
+
+## DHL
+triggers query_nowhitespace_nodash => qr/^
+                                (?:
+                                    \d{9,} |
```

Comment:
I think is should be, but I cannot find a reference on their website. I can only find this:

> The DHL Express Waybill for Time Definite shipments is a 10 digit numeric number 

But I do not now about other types of tracking numbers.